node/src/chain_spec.rs

```diff
@@ -252,7 +262,26 @@ pub fn testnet_genesis(
 		},
 		pallet_ethereum_chain_id: EthereumChainIdConfig { chain_id },
 		pallet_evm: EVMConfig {
-			accounts: BTreeMap::new(),
+			// We need _some_ code inserted at the precompile address so that
+			// the evm will actually call the address.
+			// TODO Cleanly fetch the addresses from
+			// the runtime/moonbeam precompiles and systematically fill them with code
+			// that will revert if it is called by accident (it shouldn't be because
+			// it is shadowed by the precompile).
```

Comment:
I'm not quite following this TODO... is it still relevant / should it be done before merging?

Reply:
No, we'll do it in a follow-up.

Reply:
I think a followup makes more sense. There is a design decision to be made. Basically the dilemna is this.

We need to get bogus code under every precompile address. The precompile addresses are codified in the `precompiles` crate, but the genesis config (which includes the pre-deployed bogus code) is set up here.

So the decision is whether we have the list of addresses in both places (as we do now), or do we call into the `precompiles` crate to get them somehow. I started sketching that also, but still didn't like it because even the way I sketched it the addresses were still in two places (although at least they were in the same file).